Add more info in FAQ (#264)
AY2425S1-CS2103T-F14b-3/tp: docs/UserGuide.md

**Q**: How do I transfer my data to another Computer?<br>
**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous ContactMate home folder.

**Q**: Why can't I execute certain commands like `mark S2208201I` when the displayed person list is empty after a `find` command?<br>

> **A**: Some commands that refer to an existing elderly by index or NRIC will not work if it didn't exist in the displayed person list. To execute these commands, you can use the `list` command to return to the person list view.

Reviewer: I think this might be a little clearer?
```suggestion
**A**: Commands that refer to an elderly by index or NRIC will not work if the elderly is not shown in the displayed person list. To execute these commands, you can use the `list` command to return to the full, unfiltered displayed person list view.
```

**Q**: Why are the `delete`, `edit` and `mark` commands are restricted in the history list view?<br>

> **A**: These commands are restricted in the history list view to prevent unintended modifications to an elder's details. To modify an elder's information accurately, you can use the `list` command to return to the person list view. 

Reviewer: Might be a typo for elderly

**Q**: Why can't I `mark S2208201I` when I am currently viewing the contact records of the elderly with NRIC `S2208201I`? <br>

> **A**: The `mark` command requires you to be in the person list view rather in the individual history list view. It ensures that all-elderly related commands function as expected and to avoid confusion. 

Reviewer 2: Do you think its better to stick to the way you've been writing the "`personList` view", same for the `historyList` view

Author: I think should be ok since we use **person list view** in glossary also
